serverlessworkflow/modules/ROOT/pages/index.adoc

```diff
@@ -135,83 +153,68 @@ Learn how to use {serverless_logic_web_tools_name} for creating and managing wor
 [.card]
 --
 [.card-title]
-xref:service-orchestration/orchestration-of-openapi-based-services.adoc[Orchestrating the OpenAPI services]
+xref:service-orchestration/orchestration-of-openapi-based-services.adoc[]
 [.card-description]
 Learn how to orchestrate REST services using OpenAPI specification descriptors
 --
 
 [.card]
 --
 [.card-title]
-xref:service-orchestration/orchestration-of-asyncapi-based-services.adoc[Orchestrating the AsyncAPI services]
-[.card-description]
-Learn how to trigger and consume events using AsyncAPI specification descriptors
---
-[.card]
---
-[.card-title]
-xref:service-orchestration/configuring-openapi-services-endpoints.adoc[Configuring the OpenAPI services endpoints]
+xref:service-orchestration/configuring-openapi-services-endpoints.adoc[]
 [.card-description]
 Learn how to configure OpenAPI services endpoints in different environments
 --
 
 [.card]
 --
 [.card-title]
-xref:service-orchestration/orchestration-of-grpc-services.adoc[Orchestration of gRPC based services]
+xref:service-orchestration/working-with-openapi-callbacks.adoc[]
 [.card-description]
-Learn about orchestrating gRPC services
+Learn how to use the OpenAPI Callback in your workflow application
 --
 
-[.card-section]
-== Eventing
-
 [.card]
 --
 [.card-title]
-xref:eventing/handling-events-on-workflows.adoc[Event state in {product_name}]
+xref:service-orchestration/orchestration-of-grpc-services.adoc[]
 [.card-description]
-Learn how to use the Event state in your workflow application
+Learn about orchestrating gRPC services
 --
 
-[.card]
---
-[.card-title]
-xref:eventing/working-with-callbacks.adoc[Callback state in {product_name}]
-[.card-description]
-Learn how to use the Callback state in your workflow application
---
+[.card-section]
+== Event Orchestration
 
 [.card]
 --
 [.card-title]
-xref:eventing/consume-producing-events-with-kafka.adoc[Consuming and producing events using Apache Kafka]
+xref:eventing/orchestration-of-asyncapi-based-services.adoc[]
 [.card-description]
-Learn how to configure your Serverless Workflow application to produce and consume events using Apache Kafka
+Learn how to trigger and consume events using AsyncAPI specification descriptors
 --
 
 [.card]
 --
 [.card-title]
-xref:eventing/consume-produce-events-with-knative-eventing.adoc[Consuming and producing events on Knative Eventing]
+xref:eventing/event-correlation-with-workflows.adoc[]
```

Comment:
Keep the nav and index ordered the same way :)
Please move that card after quarkus ones.

Reply:
I did that on purpose. I'll bring correlations to the front in the index instead. It's a core concept about eventing.